CI run libmamba_static_win fails silently

While waiting for the tests to run on my PR I noticed that the `libmamba_static_win` run was surprisingly quick. After checking the log I noticed that it has failed silently.

This problem is not specific to my PR, but also occurs on the master branch. This is the relevant section of the PR run of the current `HEAD` of the master branch (CI #3841, commit 57668935ce5697616d8bc1d2e1b6b0da6589a0c5):

```
2022-07-14T12:19:41.9496710Z C:\Program Files\Microsoft Visual Studio\2022\Enterprise>CALL "VC\Auxiliary\Build\vcvars64.bat" -vcvars_ver=14.16 10.0.22621.0 
2022-07-14T12:19:42.3856057Z **********************************************************************
2022-07-14T12:19:42.3869839Z ** Visual Studio 2022 Developer Command Prompt v17.2.5
2022-07-14T12:19:42.3875678Z ** Copyright (c) 2022 Microsoft Corporation
2022-07-14T12:19:42.3891993Z **********************************************************************
2022-07-14T12:19:44.5166743Z [vcvarsall.bat] Environment initialized for: 'x64'
2022-07-14T12:19:44.7382268Z CMake Warning:
2022-07-14T12:19:44.7382863Z   Ignoring extra path from command line:
2022-07-14T12:19:44.7383246Z 
2022-07-14T12:19:44.7383468Z    "\"
2022-07-14T12:19:44.7383692Z 
2022-07-14T12:19:44.7383813Z 
2022-07-14T12:19:45.1481634Z -- Building for: Visual Studio 16 2019
2022-07-14T12:19:45.1726457Z CMake Error at CMakeLists.txt:16 (project):
2022-07-14T12:19:45.1727005Z   Generator
2022-07-14T12:19:45.1727334Z 
2022-07-14T12:19:45.1727669Z     Visual Studio 16 2019
2022-07-14T12:19:45.1728365Z -- Configuring incomplete, errors occurred!
2022-07-14T12:19:45.1728867Z See also "D:/a/mamba/mamba/build/CMakeFiles/CMakeOutput.log".
2022-07-14T12:19:45.1729249Z 
2022-07-14T12:19:45.1729674Z   could not find any instance of Visual Studio.
2022-07-14T12:19:45.1729990Z 
2022-07-14T12:19:45.1730117Z 
2022-07-14T12:19:45.1730241Z 
2022-07-14T12:19:45.2739187Z '-DCMAKE_C_COMPILER_LAUNCHER' is not recognized as an internal or external command,
2022-07-14T12:19:45.2739645Z operable program or batch file.
2022-07-14T12:19:45.2801441Z '-GNinja' is not recognized as an internal or external command,
2022-07-14T12:19:45.2801836Z operable program or batch file.
2022-07-14T12:19:45.3000282Z ninja: error: loading 'build.ninja': The system cannot find the file specified.
```

The full log is here: https://github.com/mamba-org/mamba/runs/7339706232#step:5:133

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -284,8 +284,8 @@ jobs:
           cmake .. -DCMAKE_INSTALL_PREFIX=%CONDA_PREFIX%\Library ^
                    -DBUILD_LIBMAMBA=ON ^
                    -DBUILD_STATIC=ON ^
-                   -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
-                   -DCMAKE_C_COMPILER_LAUNCHER=sccache \
+                   -DCMAKE_CXX_COMPILER_LAUNCHER=sccache ^
+                   -DCMAKE_C_COMPILER_LAUNCHER=sccache ^
                    -GNinja
           ninja
           sccache --show-stats

